src/MyMind.Web/Domain/Emotions/Services/EmotionsService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MyMind.Web.Database;
+
+namespace MyMind.Web.Domain.Emotions.Services
+{
+    public class EmotionsService : IEmotionsService
+    {
+        private ISessionFactory _sessionFactory;
+
+        public EmotionsService(ISessionFactory sessionFactory)
+        {
+            this._sessionFactory = sessionFactory;
+        }
+
+        public IQueryable<Emotion> GetAllEmotions()
```

Comment:
tu bezpieczniej zwracać IEnumarable<Emotion>

IQueryable to bezpośrednie operacje na bazie i nie chcemy, żeby nasz serwis to zwracał